src/write_gcm.c

```diff
@@ -4158,6 +4158,27 @@ static int wg_write(const data_set_t *ds, const value_list_t *vl,
     }
     --queue->size;
     to_remove->next = NULL;
+    char metadata[8192];
+    metadata[0] = '{';
+    char *meta_ptr = metadata + 1;
+    int meta_size = sizeof(metadata) - 1;
```

Comment:
Thanks, Rick, for catching the bug. Thanks, Corey, for suggesting `bufprintf` (lunch sounds good). Done.